ios/Headers/threshold.h

```diff
@@ -0,0 +1,189 @@
+// COPIED FROM https://github.com/celo-org/celo-threshold-bls-rs/blob/master/crates/threshold-bls-ffi/cross/threshold.h
+// DO NOT MODIFY DIRECTLY
+
+#include <stdarg.h>
```

Comment:
No need to review this file, it's copied from another repo as mentioned in the comment above